src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+type User = {
+  user: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  };
+};
```

Comment:
```suggestion
type Props = {
  user: User;
};
```
create the User type